third_party/nacl_sdk/nacl_sdk/sdk_tools/cacerts.txt

```diff
@@ -1,77 +1,22 @@
-# Copyright (c) 2012 The Chromium Authors. All rights reserved.
-# Use of this source code is governed by a BSD-style license that can be
-# found in the LICENSE file.
-
-# The following SSL certificates are used to validate the SSL connection
```

Comment:
nit: I guess this comment here is still valid? Also the cert name/identifier, although it's just one now.

Reply:
Done, good point.